### ISSUE TITLE
Update demo page with latest version

### ISSUE THREAD

**Issue by SomeKittens** [angular-ui/ui-layout#119]
Demo page ( http://angular-ui.github.io/ui-layout/ ) has an old version of ui-layout

**Comment by SomeKittens** [comment]
Make sure the demo plunkers are pointing at the latest tag (`v1.2.2` or whichever) rather than just `master` so that a breaking change in the future doesn't break the page/Plunkers

**Comment by theclapp** [comment]
The demo plunker I built today (9/16/15), based on the demo, now points to the correct tag (v0.0.2).  See #81, and here: http://plnkr.co/edit/mxbTJO?p=preview.

**Comment by wu-lee** [comment]
@theclapp Tried that, but the link http://plnkr.co/edit/mxbTJO?p=preview doesn't work for me (Firefox 41.0.1 on Ubuntu Precise).  There is a frame and a grey empty content pane. If I open the javascript console I see an error:

    SecurityError: The operation is insecure.
    ...ion(t){var i,r,a,c,u,h;for(a=o.currentVer!==t.ver.semver?l.obsolete:l.current,0>...
    editor-0.7.84.js (line 6)

This is the same as I get if I follow the link on the demo page http://plnkr.co/zB4mhgJyVz7GlwG8JmeM and then click the "Launch in Editor" button.  

Actually, even before clicking the button, although the page appears to work there is an error.

    TypeError: href is null
    n/<()landing-0.7.84.js (line 2)
    Ft/this.$get</<()landing-0.7.84.js (line 1)
    Ot/this.$get</V.prototype.$set/<()landing-0.7.84.js (line 1)
    i()landing-0.7.84.js (line 1)
    Ot/this.$get</V.prototype.$set()landing-0.7.84.js (line 1)
    [... rest of long and unhelpful stack trace snipped ...]
    stackFrame.js (line 357)

This "TypeError:href is null" suggests an internal exception is being masked by angularjs.

This link, and the one from the home page appear to work in Chromium (45.0.2454.101 Ubuntu 14.04 (64-bit)) but again, there appear to be a lot of errors in the console, more than I can list here, but for example:

    GET https://cdn.rawgit.com/jpillora/xdomain/0.7.3/dist/xdomain.min.js net::ERR_BLOCKED_BY_CLIENT
    plnkr.co/:36 GET http://s7.addthis.com/js/300/addthis_widget.js net::ERR_BLOCKED_BY_CLIENT
    ?p=preview:6 Uncaught ReferenceError: xdomain is not defined
    editor-0.7.84.js:2 URL visited /?p=preview
    ?p=preview:28 GET http://www.google-analytics.com/ga.js net::ERR_BLOCKED_BY_CLIENT
    ?p=preview:32 GET http://www.google-analytics.com/analytics.js net::ERR_BLOCKED_BY_CLIENT
    editor-0.7.84.js:16 Automatically scrolling cursor into view after selection change this will be disabled in the next version set editor.$blockScrolling = Infinity to disable this message
    5226171f87bc6890da0000a0.js:14 GET http://pixel-geo.prfct.co/tagjs?a_id=3828&source=js_tag net::ERR_BLOCKED_BY_CLIENT

**Comment by theclapp** [comment]
I'm not sure what to suggest.  The Plunkr I listed still works for me, in Chrome 46.0.2490.71 m on Windows 7.

I do not get as many BLOCKED_BY_CLIENT errors as you, just one, which appears to be the work of AdBlock Plus.

If you have something like that running, disable it on that site and see if that helps ... ?  Aside from that, I dunno.

**Comment by wu-lee** [comment]
Yes, I think the BLOCKED_BY_CLIENT stuff may be Privacy Badger. I do understand you're not an author, just another commenter, this is a general comment intended to suggest something may still be wrong.  It could also just be my set-up, I've not gotten to the bottom of it yet!  This plus the fact that the demo doesn't seem to match up with the (currently scant) documentation does make it  confusing for newcomers... although I seem to have gotten it working with some experimentation.

**Comment by SomeKittens** [comment]
The latest version of UI-Layout is `v1.3.1`
